manifests/init.pp

```diff
@@ -247,6 +249,18 @@
     }
   }
 
+  case $::operatingsystem {
+    'Ubuntu': {
+      if $lsbmajdistrelease >= 14 {
```

Comment:
I would suggest go back to the fact osversion and use versioncmp function to compare